### ISSUE TITLE
Block Outgoing communication from TopologyManager during startup

### ISSUE THREAD

**Issue by pihme** [camunda-cloud/zeebe#7543]
**Description**

During startup partitions may be started in one Broker before the command API and subscription API are ready to service request for them. To prevent them from receiving traffic before the broker can service traffic for a given partition, we need to block any outbound communication from `TopologyManager` with respect to the not yet ready partition. Secondly, once the broker is ready to service a partition we need to trigger `TopolgyManager` to announce the partition to the rest of the cluster.

(Ordinarily, the `TopologyManager` acts as a listener and broadcasts changes regardless of whether the broker is ready to service traffic for the partition)

**Comment by pihme** [comment]
No longer needed as topology manager is now started after command API and subscription API
